x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -636,6 +667,28 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
       }
     }
 
+    if (props.sourceIndicesWithGeoFields[anomaly.jobId]) {
+      items.push(
```

Comment:
Might be related to the functional test failure, but if I click on the actions menu cog icon in the anomalies table in the Single Metric Viewer, the page crashes, with these errors in the console:

<img width="665" alt="image" src="https://user-images.githubusercontent.com/7405507/186407181-658b2529-5cfb-48bc-9923-e49ab04c04b3.png">

This action should be present in the table in the Single Metric Viewer.

Reply:
👍 I'll add it in - we don't currently have any maps integration stuff in the single metric viewer so I didn't think we'd add it.

Reply:
Added link in Single Metric viewer in 68e56386c7785f7a02c738426a084043212172c6